pkg/ensurance/collector/types/interface.go

```diff
@@ -1,11 +1,11 @@
-package collector
+package types
```

Comment:
I prefer the previous way, interface should be in the upper folder of implementer, please check k8s code for reference.

Reply:
done~